<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="SOOSANui-MacBookAir.local" failures="0" tests="39" name="eWalker_SSL_VPN_V10.eWalker_SSL_VPN_Android" time="20.520" errors="1" timestamp="2023-05-24T13:57:45 KST" skipped="38">
  <testcase classname="eWalker_SSL_VPN_V10.eWalker_SSL_VPN_Android" name="R06_T03" time="0.000">
    <skipped/>
  </testcase> <!-- R06_T03 -->
  <system-out/>
  <testcase classname="eWalker_SSL_VPN_V10.eWalker_SSL_VPN_Android" name="R07_T05" time="0.000">
    <skipped/>
  </testcase> <!-- R07_T05 -->
  <system-out/>
  <testcase classname="eWalker_SSL_VPN_V10.eWalker_SSL_VPN_Android" name="R06_T01" time="0.000">
    <skipped/>
  </testcase> <!-- R06_T01 -->
  <system-out/>
  <testcase classname="eWalker_SSL_VPN_V10.eWalker_SSL_VPN_Android" name="R06_T05" time="0.000">
    <skipped/>
  </testcase> <!-- R06_T05 -->
  <system-out/>
  <testcase classname="eWalker_SSL_VPN_V10.eWalker_SSL_VPN_Android" name="R09_T03" time="0.000">
    <skipped/>
  </testcase> <!-- R09_T03 -->
  <system-out/>
  <testcase classname="eWalker_SSL_VPN_V10.eWalker_SSL_VPN_Android" name="R02_T02" time="0.000">
    <skipped/>
  </testcase> <!-- R02_T02 -->
  <system-out/>
  <testcase classname="eWalker_SSL_VPN_V10.eWalker_SSL_VPN_Android" name="R04_T06" time="0.000">
    <skipped/>
  </testcase> <!-- R04_T06 -->
  <system-out/>
  <testcase classname="eWalker_SSL_VPN_V10.eWalker_SSL_VPN_Android" name="R06_T02" time="0.000">
    <skipped/>
  </testcase> <!-- R06_T02 -->
  <system-out/>
  <testcase classname="eWalker_SSL_VPN_V10.eWalker_SSL_VPN_Android" name="R03_T01" time="0.000">
    <skipped/>
  </testcase> <!-- R03_T01 -->
  <system-out/>
  <testcase classname="eWalker_SSL_VPN_V10.eWalker_SSL_VPN_Android" name="R09_T01" time="0.000">
    <skipped/>
  </testcase> <!-- R09_T01 -->
  <system-out/>
  <testcase classname="eWalker_SSL_VPN_V10.eWalker_SSL_VPN_Android" name="R10_T03" time="0.000">
    <skipped/>
  </testcase> <!-- R10_T03 -->
  <system-out/>
  <testcase classname="eWalker_SSL_VPN_V10.eWalker_SSL_VPN_Android" name="R11_T02" time="0.000">
    <skipped/>
  </testcase> <!-- R11_T02 -->
  <system-out/>
  <testcase classname="eWalker_SSL_VPN_V10.eWalker_SSL_VPN_Android" name="R04_T03" time="0.000">
    <skipped/>
  </testcase> <!-- R04_T03 -->
  <system-out/>
  <testcase classname="eWalker_SSL_VPN_V10.eWalker_SSL_VPN_Android" name="R04_T05" time="0.000">
    <skipped/>
  </testcase> <!-- R04_T05 -->
  <system-out/>
  <testcase classname="eWalker_SSL_VPN_V10.eWalker_SSL_VPN_Android" name="R10_T05" time="0.000">
    <skipped/>
  </testcase> <!-- R10_T05 -->
  <system-out/>
  <testcase classname="eWalker_SSL_VPN_V10.eWalker_SSL_VPN_Android" name="R08_T04" time="0.000">
    <skipped/>
  </testcase> <!-- R08_T04 -->
  <system-out/>
  <testcase classname="eWalker_SSL_VPN_V10.eWalker_SSL_VPN_Android" name="R09_T04" time="0.000">
    <skipped/>
  </testcase> <!-- R09_T04 -->
  <system-out/>
  <testcase classname="eWalker_SSL_VPN_V10.eWalker_SSL_VPN_Android" name="R10_T01" time="0.000">
    <skipped/>
  </testcase> <!-- R10_T01 -->
  <system-out/>
  <testcase classname="eWalker_SSL_VPN_V10.eWalker_SSL_VPN_Android" name="R11_T01" time="0.000">
    <skipped/>
  </testcase> <!-- R11_T01 -->
  <system-out/>
  <testcase classname="eWalker_SSL_VPN_V10.eWalker_SSL_VPN_Android" name="R08_T05" time="0.000">
    <skipped/>
  </testcase> <!-- R08_T05 -->
  <system-out/>
  <testcase classname="eWalker_SSL_VPN_V10.eWalker_SSL_VPN_Android" name="R09_T05" time="0.000">
    <skipped/>
  </testcase> <!-- R09_T05 -->
  <system-out/>
  <testcase classname="eWalker_SSL_VPN_V10.eWalker_SSL_VPN_Android" name="R07_T04" time="0.000">
    <skipped/>
  </testcase> <!-- R07_T04 -->
  <system-out/>
  <testcase classname="eWalker_SSL_VPN_V10.eWalker_SSL_VPN_Android" name="beforeTest" time="20.520">
    <error message="Unable to create a new remote session. Please check the server log for more details. Original error: An unknown server-side error occurred while processing the command. Original error: Could not find a connected Android device in 20118ms.
Build info: version: &#039;3.141.59&#039;, revision: &#039;e82be7d358&#039;, time: &#039;2018-11-14T08:17:03&#039;
System info: host: &#039;SOOSANui-MacBookAir.local&#039;, ip: &#039;fe80:0:0:0:858:5171:75cc:cdd9%en0&#039;, os.name: &#039;Mac OS X&#039;, os.arch: &#039;aarch64&#039;, os.version: &#039;13.3.1&#039;, java.version: &#039;17.0.6&#039;
Driver info: driver.version: AndroidDriver
remote stacktrace: UnknownError: An unknown server-side error occurred while processing the command. Original error: Could not find a connected Android device in 20118ms.
    at getResponseForW3CError (/Applications/Appium Server GUI.app/Contents/Resources/app/node_modules/appium/node_modules/appium-base-driver/lib/protocol/errors.js:804:9)
    at asyncHandler (/Applications/Appium Server GUI.app/Contents/Resources/app/node_modules/appium/node_modules/appium-base-driver/lib/protocol/protocol.js:380:37)
Build info: version: &#039;3.141.59&#039;, revision: &#039;e82be7d358&#039;, time: &#039;2018-11-14T08:17:03&#039;
System info: host: &#039;SOOSANui-MacBookAir.local&#039;, ip: &#039;fe80:0:0:0:858:5171:75cc:cdd9%en0&#039;, os.name: &#039;Mac OS X&#039;, os.arch: &#039;aarch64&#039;, os.version: &#039;13.3.1&#039;, java.version: &#039;17.0.6&#039;
Driver info: driver.version: AndroidDriver" type="org.openqa.selenium.SessionNotCreatedException">
      <![CDATA[org.openqa.selenium.SessionNotCreatedException: Unable to create a new remote session. Please check the server log for more details. Original error: An unknown server-side error occurred while processing the command. Original error: Could not find a connected Android device in 20118ms.
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
System info: host: 'SOOSANui-MacBookAir.local', ip: 'fe80:0:0:0:858:5171:75cc:cdd9%en0', os.name: 'Mac OS X', os.arch: 'aarch64', os.version: '13.3.1', java.version: '17.0.6'
Driver info: driver.version: AndroidDriver
remote stacktrace: UnknownError: An unknown server-side error occurred while processing the command. Original error: Could not find a connected Android device in 20118ms.
    at getResponseForW3CError (/Applications/Appium Server GUI.app/Contents/Resources/app/node_modules/appium/node_modules/appium-base-driver/lib/protocol/errors.js:804:9)
    at asyncHandler (/Applications/Appium Server GUI.app/Contents/Resources/app/node_modules/appium/node_modules/appium-base-driver/lib/protocol/protocol.js:380:37)
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
System info: host: 'SOOSANui-MacBookAir.local', ip: 'fe80:0:0:0:858:5171:75cc:cdd9%en0', os.name: 'Mac OS X', os.arch: 'aarch64', os.version: '13.3.1', java.version: '17.0.6'
Driver info: driver.version: AndroidDriver
at io.appium.java_client.remote.AppiumCommandExecutor$1.createSession(AppiumCommandExecutor.java:208)
at io.appium.java_client.remote.AppiumCommandExecutor.createSession(AppiumCommandExecutor.java:217)
at io.appium.java_client.remote.AppiumCommandExecutor.execute(AppiumCommandExecutor.java:239)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:552)
at io.appium.java_client.DefaultGenericMobileDriver.execute(DefaultGenericMobileDriver.java:41)
at io.appium.java_client.AppiumDriver.execute(AppiumDriver.java:1)
at io.appium.java_client.android.AndroidDriver.execute(AndroidDriver.java:1)
at org.openqa.selenium.remote.RemoteWebDriver.startSession(RemoteWebDriver.java:213)
at io.appium.java_client.AppiumDriver.startSession(AppiumDriver.java:336)
at org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:131)
at io.appium.java_client.DefaultGenericMobileDriver.<init>(DefaultGenericMobileDriver.java:37)
at io.appium.java_client.AppiumDriver.<init>(AppiumDriver.java:88)
at io.appium.java_client.AppiumDriver.<init>(AppiumDriver.java:98)
at io.appium.java_client.android.AndroidDriver.<init>(AndroidDriver.java:94)
at eWalker_SSL_VPN_V10.eWalker_SSL_VPN_Android.beforeTest(eWalker_SSL_VPN_Android.java:2189)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:61)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:366)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:320)
at org.testng.TestRunner.invokeTestConfigurations(TestRunner.java:617)
at org.testng.TestRunner.beforeRun(TestRunner.java:607)
at org.testng.TestRunner.run(TestRunner.java:578)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: java.lang.reflect.InvocationTargetException
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at io.appium.java_client.remote.AppiumCommandExecutor$1.createSession(AppiumCommandExecutor.java:186)
... 38 more
Caused by: org.openqa.selenium.WebDriverException: An unknown server-side error occurred while processing the command. Original error: Could not find a connected Android device in 20118ms.
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
System info: host: 'SOOSANui-MacBookAir.local', ip: 'fe80:0:0:0:858:5171:75cc:cdd9%en0', os.name: 'Mac OS X', os.arch: 'aarch64', os.version: '13.3.1', java.version: '17.0.6'
Driver info: driver.version: AndroidDriver
remote stacktrace: UnknownError: An unknown server-side error occurred while processing the command. Original error: Could not find a connected Android device in 20118ms.
    at getResponseForW3CError (/Applications/Appium Server GUI.app/Contents/Resources/app/node_modules/appium/node_modules/appium-base-driver/lib/protocol/errors.js:804:9)
    at asyncHandler (/Applications/Appium Server GUI.app/Contents/Resources/app/node_modules/appium/node_modules/appium-base-driver/lib/protocol/protocol.js:380:37)
at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499)
at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:480)
at org.openqa.selenium.remote.W3CHandshakeResponse.lambda$errorHandler$0(W3CHandshakeResponse.java:62)
at org.openqa.selenium.remote.HandshakeResponse.lambda$getResponseFunction$0(HandshakeResponse.java:30)
at org.openqa.selenium.remote.ProtocolHandshake.lambda$createSession$0(ProtocolHandshake.java:126)
at java.base/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:197)
at java.base/java.util.Spliterators$ArraySpliterator.tryAdvance(Spliterators.java:1002)
at java.base/java.util.stream.ReferencePipeline.forEachWithCancel(ReferencePipeline.java:129)
at java.base/java.util.stream.AbstractPipeline.copyIntoWithCancel(AbstractPipeline.java:527)
at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:513)
at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:499)
at java.base/java.util.stream.FindOps$FindOp.evaluateSequential(FindOps.java:150)
at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
at java.base/java.util.stream.ReferencePipeline.findFirst(ReferencePipeline.java:647)
at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:128)
... 43 more
]]>
    </error>
  </testcase> <!-- beforeTest -->
  <system-out/>
  <testcase classname="eWalker_SSL_VPN_V10.eWalker_SSL_VPN_Android" name="R07_T01" time="0.000">
    <skipped/>
  </testcase> <!-- R07_T01 -->
  <system-out/>
  <testcase classname="eWalker_SSL_VPN_V10.eWalker_SSL_VPN_Android" name="R08_T03" time="0.000">
    <skipped/>
  </testcase> <!-- R08_T03 -->
  <system-out/>
  <testcase classname="eWalker_SSL_VPN_V10.eWalker_SSL_VPN_Android" name="R04_T09" time="0.000">
    <skipped/>
  </testcase> <!-- R04_T09 -->
  <system-out/>
  <testcase classname="eWalker_SSL_VPN_V10.eWalker_SSL_VPN_Android" name="R03_T02" time="0.000">
    <skipped/>
  </testcase> <!-- R03_T02 -->
  <system-out/>
  <testcase classname="eWalker_SSL_VPN_V10.eWalker_SSL_VPN_Android" name="R08_T01" time="0.000">
    <skipped/>
  </testcase> <!-- R08_T01 -->
  <system-out/>
  <testcase classname="eWalker_SSL_VPN_V10.eWalker_SSL_VPN_Android" name="R04_T02" time="0.000">
    <skipped/>
  </testcase> <!-- R04_T02 -->
  <system-out/>
  <testcase classname="eWalker_SSL_VPN_V10.eWalker_SSL_VPN_Android" name="R04_T07" time="0.000">
    <skipped/>
  </testcase> <!-- R04_T07 -->
  <system-out/>
  <testcase classname="eWalker_SSL_VPN_V10.eWalker_SSL_VPN_Android" name="R06_T06" time="0.000">
    <skipped/>
  </testcase> <!-- R06_T06 -->
  <system-out/>
  <testcase classname="eWalker_SSL_VPN_V10.eWalker_SSL_VPN_Android" name="R04_T01" time="0.000">
    <skipped/>
  </testcase> <!-- R04_T01 -->
  <system-out/>
  <testcase classname="eWalker_SSL_VPN_V10.eWalker_SSL_VPN_Android" name="R10_T04" time="0.000">
    <skipped/>
  </testcase> <!-- R10_T04 -->
  <system-out/>
  <testcase classname="eWalker_SSL_VPN_V10.eWalker_SSL_VPN_Android" name="R04_T08" time="0.000">
    <skipped/>
  </testcase> <!-- R04_T08 -->
  <system-out/>
  <testcase classname="eWalker_SSL_VPN_V10.eWalker_SSL_VPN_Android" name="R02_T03" time="0.000">
    <skipped/>
  </testcase> <!-- R02_T03 -->
  <system-out/>
  <testcase classname="eWalker_SSL_VPN_V10.eWalker_SSL_VPN_Android" name="R02_T01" time="0.000">
    <skipped/>
  </testcase> <!-- R02_T01 -->
  <system-out/>
  <testcase classname="eWalker_SSL_VPN_V10.eWalker_SSL_VPN_Android" name="R04_T04" time="0.000">
    <skipped/>
  </testcase> <!-- R04_T04 -->
  <system-out/>
  <testcase classname="eWalker_SSL_VPN_V10.eWalker_SSL_VPN_Android" name="R07_T03" time="0.000">
    <skipped/>
  </testcase> <!-- R07_T03 -->
  <system-out/>
  <testcase classname="eWalker_SSL_VPN_V10.eWalker_SSL_VPN_Android" name="R01_T01" time="0.000">
    <skipped/>
  </testcase> <!-- R01_T01 -->
  <system-out/>
</testsuite> <!-- eWalker_SSL_VPN_V10.eWalker_SSL_VPN_Android -->
